graylog2-web-interface/src/views/pages/DashboardsPage.tsx

```diff
@@ -84,10 +98,14 @@ const DashboardsPage = () => {
 
         <Row className="content">
           <Col md={12}>
-            <DashboardList dashboards={list}
-                           pagination={pagination}
-                           handleSearch={handleSearch}
-                           handleDashboardDelete={handleDashboardDelete} />
+            {list !== undefined && list.length === 0 && !searchQuery
```

Comment:
It is a nitpick, but you could simplify this with `!!list?.length && !searchQuery`